FE/src/components/Histories.js

```diff
@@ -0,0 +1,40 @@
+import HistoryCard from "./HistoryCard.js";
+import { store } from "../store.js";
+import { GET_HISTORY } from "../dummyData.js";
+import { $ } from "../utils.js";
+
+export default class Histories {
+  constructor(target) {
+    this.target = target;
+    this.historyCardComponents = GET_HISTORY.map(
+      (history) => new HistoryCard(history)
+    );
+    store.initState("history");
+    store.subscribe("history", this.render.bind(this));
+    store.setState("history", GET_HISTORY);
+  }
+  template() {
+    return `
+        <button class="history__btn--close">x</button>
+        ${this.historyCardComponents.reduce(
+          (prev, h) => prev + h.template(),
```

Comment:
변수명을 가급적이면 줄이지 않는게 코드를 이해하는데 용이합니다 
```suggestion
          (prev, historyCard) => prev + historyCard.template(),
```